shared/tests/utility/math/docs/cartesianToSpherical.m

```diff
@@ -0,0 +1,17 @@
+function conversion = cartesianToSpherical(inputVec)
```

Comment:
Ahh, there has been a miscommunication from my end! When I was saying we want to use matlab for our ground truth, what I meant was we should use the matlab builtin functions which do these. So instead of implementing a function which does the same thing on matlab, use their standard library versions [(here's this example)](https://au.mathworks.com/help/matlab/ref/sph2cart.html) for these functions.
Our reasoning is that matlab should certainly have rock solid implementations for these functions (because they're common functions), so we want to match _their_ implementation closely.

My bad for poorly communicating that to you!

Reply:
Also, the scripts that remain should be placed in a subfolder of this one I think

Reply:
That makes sense! Shouldn't be a big change. I'll sort that out.

Reply:
For the record, pretty sure you did say "built in functions" but i was thinking of trig functions.

Reply:
Is there a Python implementation of these functions? Considering the majority of our code base is C++/Python it would be nicer if we could stick to those languages as much as possible

Reply:
I have found a python implementation in `astropy` - I think it uses another convention though. That means the results it gives would need to be permuted before we used them, which is fine, but certainly worth noting if that's what you think we should go for.